merkle-tree/src/web3.js

```diff
@@ -21,9 +24,9 @@ export default {
 
     logger.info('Blockchain Connecting ...');
     const provider = new Web3.providers.WebsocketProvider(
-      `${config.web3.host}:${config.web3.port}`,
-      null,
-      config.web3.options,
+      this.buildUrl(),
+      // null,
+      // config.web3.options,
```

Comment:
It might be important to pass in the config's web3 options in some cases, to allow devs to specify stuff like default gas, number of blocks before a tx is considered final, etc. Perhaps the options could be uncommented?